test/fw/ptl/utils/pbs_testsuite.py

```diff
@@ -516,6 +516,13 @@ def setUp(self):
             self.revert_pbsconf()
             self.revert_schedulers()
             self.revert_moms()
+
+        # turn off opt_backfill_fuzzy
+        a = {'opt_backfill_fuzzy': 'off'}
+        for schedinfo in self.schedulers.values():
+            for schedname in schedinfo.keys():
+                self.server.manager(MGR_CMD_SET, SCHED, a, id=schedname)
+
```

Comment:
Why are you turning off opt_backfill_fuzzy?  Plus, the reason I had to ask this means your comment is not helpful.  Either add why or remove it.

Reply:
I am fine with this.  Make sure the opt_backfill_fuzzy test still passes.  It might have a test which tests the default behavior.

Reply:
Thanks, I ran a regression after this and TestCalendaring passed. Let me know if you'd like me to run any other test as well.

Reply:
Is that where the opt_backfill_fuzzy tests are?  I thought there was a test plan just for it.

Reply:
that's all that i found. There were a couple in other test suites as well, but that was it.